### PR TITLE
Upgrade packages to cosmwasm 0.15.0

### PR DESCRIPTION
**Pull request by uint**

### CODE REVIEW

File: packages/controllers/src/claim.rs [hunk 1]
```diff
@@ -62,7 +62,7 @@ impl<'a> Claims<'a> {
         block: &BlockInfo,
         cap: Option<Uint128>,
     ) -> StdResult<Uint128> {
-        let mut to_send = Uint128(0);
+        let mut to_send = Uint128::new(0);
```
Comment (orkunkl) [inline packages/controllers/src/claim.rs:65]:
```suggestion
        let mut to_send = Uint128::zero();
```

File: packages/cw0/src/balance.rs [hunk 2]
```diff
@@ -65,7 +65,7 @@ impl NativeBalance {
     }
 
     pub fn is_empty(&self) -> bool {
-        !self.0.iter().any(|x| x.amount != Uint128(0))
+        !self.0.iter().any(|x| x.amount != Uint128::new(0))
```
Comment (orkunkl) [inline packages/cw0/src/balance.rs:68]:
```suggestion
        !self.0.iter().any(|x| x.amount != Uint128::zero())
```

File: packages/cw0/src/payment.rs [hunk 3]
```diff
@@ -42,7 +42,7 @@ pub fn must_pay(info: &MessageInfo, denom: &str) -> Result<Uint128, PaymentError
 /// denom was sent. Otherwise, returns the amount of `denom` sent, or 0 if nothing sent.
 pub fn may_pay(info: &MessageInfo, denom: &str) -> Result<Uint128, PaymentError> {
     if info.funds.is_empty() {
-        Ok(Uint128(0))
+        Ok(Uint128::new(0))
```
Comment (orkunkl) [inline packages/cw0/src/payment.rs:45]:
```suggestion
        Ok(Uint128::zero())
```

File: packages/cw0/src/payment.rs [hunk 4]
```diff
@@ -97,10 +97,10 @@ mod test {
         let mixed_payment = mock_info(SENDER, &[coin(50, atom), coin(120, "wei")]);
 
         let res = may_pay(&no_payment, atom).unwrap();
-        assert_eq!(res, Uint128(0));
+        assert_eq!(res, Uint128::new(0));
```
Comment (orkunkl) [inline packages/cw0/src/payment.rs:100]: better to use `zero()` as above

**Review by orkunkl — APPROVED** [review]
Simple change, rest LGTM

**Comment by uint** [comment]
Can we merge with CI failing?

**Comment by orkunkl** [comment]
> Can we merge with CI failing?

Yep

**Pull request merged**